pyvips/vimage.py

```diff
@@ -324,11 +324,11 @@ def __exit__(self, type, value, traceback):
     def __getitem__(self, arg):
         if isinstance(arg, slice):
             i = 0
-            if arg.start != None:
+            if arg.start is not None:
```

Comment:
Is this a style change, or did you find a bug? It sounds like a good change anyway.